uv.lock

```diff
@@ -0,0 +1,367 @@
+version = 1
+revision = 1
+requires-python = ">=3.11, <3.14"
+
+[[package]]
+name = "anyio"
+version = "4.7.0"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "idna" },
+    { name = "sniffio" },
+    { name = "typing-extensions", marker = "python_full_version < '3.13'" },
+]
```

Comment:
Is there an issue if Python 3.13 is used since marker = "python_full_version < '3.13'"´for the dependency "typing-extension"

Reply:
tox executes unittest under python 3.13. I guess it works